oe_theme.theme

```diff
@@ -1706,6 +1713,36 @@ function oe_theme_preprocess_oe_corporate_blocks_eu_footer(array &$variables) {
     $site_logo_href .= '/index_' . EuropeanUnionLanguages::getInternalLanguageCode($variables['current_language_id']);
   }
   $variables['path'] = $site_logo_href;
+
+  _oe_theme_preprocess_site_specific_footer_links($variables);
+}
+
+/**
+ * Helper function to preprocess footer links and set external flag to them.
+ *
+ * @param array $variables
+ *   The variables array from the preprocess hook.
+ */
+function _oe_theme_preprocess_site_specific_footer_links(array &$variables): void {
+  if (empty($variables['site_specific_footer']['other_links'])) {
+    return;
+  }
+
+  foreach ($variables['site_specific_footer']['other_links'] as $section => $links) {
+    if (empty($links['links'])) {
+      continue;
+    }
+
+    foreach ($links['links'] as $index => $link) {
+      if (!$link['href'] instanceof Url) {
+        continue;
+      }
+
+      if ($link['href']->isExternal()) {
```

Comment:
We need to consider links under `europa.eu` domain ([reqs page](https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=550930772)) as internal so we shouldn't add the external icon.

<img width="1236" alt="europa eu" src="https://user-images.githubusercontent.com/22908988/160290520-a8449753-98fa-45e7-b513-f0de29f6d63b.png">